src/kOS/Suffixed/BodyAtmosphere.cs

```diff
@@ -14,9 +15,10 @@ public BodyAtmosphere(CelestialBody celestialBody)
             AddSuffix("BODY", new Suffix<string>(()=> celestialBody.bodyName));
             AddSuffix("EXISTS", new Suffix<bool>(()=> celestialBody.atmosphere));
             AddSuffix("OXYGEN", new Suffix<bool>(()=> celestialBody.atmosphere && celestialBody.atmosphereContainsOxygen));
-            AddSuffix("SCALE", new Suffix<double>(()=> celestialBody.atmosphere ? celestialBody.atmosphereScaleHeight : 0));
-            AddSuffix("SEALEVELPRESSURE", new Suffix<double>(()=> celestialBody.atmosphere ? celestialBody.staticPressureASL : 0));
-            AddSuffix("HEIGHT", new Suffix<double>(()=> celestialBody.atmosphere ? celestialBody.maxAtmosphereAltitude : 0));
+            AddSuffix("SEALEVELPRESSURE", new Suffix<double>(()=> celestialBody.atmosphere ? celestialBody.atmospherePressureSeaLevel : 0));
+            AddSuffix("HEIGHT", new Suffix<double>(()=> celestialBody.atmosphere ? celestialBody.atmosphereDepth : 0));
+
+            AddSuffix("SCALE", new Suffix<double>(() => { throw new KOSAtmosphereDeprecationException("0.17.2","SCALE","<None>",string.Empty); }));
```

Comment:
a simple variable scale no longer works, it has been replaced with both temp and height as variables.